src/guess-game.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { useState } from "react";
+
+export default function GuessGame(props) {
+    const { cheatProbability } = props;
+    const [guess, setGuess] = useState(0);
+    const targetNumber = Math.floor(Math.random() * 1000) + 1;
+    const [target] = useState(targetNumber);
+    const [cheat] = useState(Math.random);
```

Comment:
If it is a variable without a setter there is no reason to use a state for it.
The reason to have a state is so that the component will know to update itself when data changes.
@ynonp will probably go over this again at some point, this is called the "React component life-cycle"
If it is a static value, it will not trigger any change and re-render, so just define a `const`.